index.html

```diff
@@ -259,6 +259,25 @@ <h1>Important Terms</h1>
 			  <p>Operating systems and other platforms provide a set of interfaces that expose information about <a class="termref">objects</a> and <a class="termref">events</a> to <a>assistive technologies</a>. Assistive technologies use these interfaces to get information about and interact with those <a class="termref">widgets</a>. Examples of accessibility APIs are <a href="https://docs.microsoft.com/en-us/windows/win32/winauto/microsoft-active-accessibility">Microsoft Active Accessibility</a> [[MSAA]], <a href="https://docs.microsoft.com/en-us/windows/win32/winauto/entry-uiauto-win32">Microsoft User Interface Automation</a> [[UI-AUTOMATION]], <abbr title="Microsoft Active Accessibility">MSAA</abbr> with <cite><a href="https://docs.microsoft.com/en-us/windows/win32/winauto/iaccessibleex"><abbr title="User Interface Automation">UIA</abbr> Express</a></cite> [[UIA-EXPRESS]], the
 				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT-SPI]], and <a href="https://wiki.linuxfoundation.org/accessibility/iaccessible2/start">IAccessible2</a> [[IAccessible2]].</p>
 			</dd>
+			<dt><dfn data-export="" data-lt="accessibility child|accessibility children|child element|child|children|child elements">Accessibility child</dfn></dt>
+			<dd>
+				<p>To calculate the <a>accessibility children</a> of an element `X`, first considered a modified <abbr title="Document Object Model">DOM</abbr> tree where elements (and their subtrees) specified by `aria-owns` are reparented to the element with `aria-owns` set.
```

Comment:
The description of “a modified DOM tree” threw me for a loop here. Would it be more accurate to name the “[accessibility tree](https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1965.html#accessibility_tree)” instead, since it’s generated from the DOM but then modified through features like `aria-owns`, `aria-hidden`, thereby diverging from the actual DOM? That would also feel harmonious with the “accessibility child/descendant/parent” terminology, IMO.

Reply:
hmmm yeah honestly I thought of that but then @smhigley thought we were building up to define the accessibility tree in terms of accessibility child/parent. we have a bit of a chicken and an egg situation